tests/sandbox/testCases/updateMultipleStatys_testCases.ts

```diff
@@ -0,0 +1,126 @@
+import { PatchMessageRequestBody } from "../../component-tests/apiGateway-tests/testCases/UpdateLetterStatus";
+import { RequestSandBoxHeaders, sandBoxHeader } from "../../constants/request_headers";
+
+export type ApiSandboxUpdateLetterStatusTestData = {
+  testCase: string;
+  header: RequestSandBoxHeaders;
+  body: PostMessageRequestBody;
+  expectedStatus: number;
+};
+
+type PostMessageRequestBody = {
+    data: postRequest []
+}
+
+type postRequest = {
+    type: string;
+    id: string;
+    attributes: {
+      reasonCode?: number;
+      reasonText?: string;
+      status: string;
+    }
+};
+
+export const apiSandboxMultipleLetterStatusTestData: ApiSandboxUpdateLetterStatusTestData[] =
+[{
+    testCase: '200 response if records are updated',
```

Comment:
The request and response for this endpoint has changed to make reasonCode a string.
You can get the latest one from sandbox/data